bin/migration/tables/cases.py

```diff
@@ -0,0 +1,103 @@
+from .helpers import check_existing_record, parse_to_timestamp, audit_entry_creation, log_failed_imports
+import uuid
+
+class CaseManager:
+    def __init__(self, source_cursor):
+        self.source_cursor = source_cursor
+        self.failed_imports = set()
+
+    def get_data(self):
+        self.source_cursor.execute("SELECT * FROM public.cases")
+        return self.source_cursor.fetchall()
```

Comment:
@mazzopardi2 there is a 'recordingstatus' field available at the existing cases table. If the case is deleted then the 'deleted' status being entered at this field. Have you migrated this value?

Reply:
Recording status has been moved into the capture session table. It is an enum value that can be one of: `STANDBY`, `INITIALIZATION`, `RECORDING`, `FINISHED`, `PROCESSING`, `AVAILABLE` or `FUTURE`. @JeyaSolirius Are these values similar to what are currently being stored in the cases table's recording status column at the moment?

Since there is not a deleted status anymore, you might need to check when it was marked as deleted (the audit table might have a timestamp) and set the `deleted_at` column. This `deleted_at` timestamp might also need to be replicated in the booking and capture session tables- but I'm unsure of this.

Also, @mazzopardi2 this is probably where you get the value for recording status for the capture session table.

Reply:
Sorry, @lucas-phillips28  & @JeyaSolirius  - I didn't see these messages - will look into that now

Reply:
@mazzopardi2 why 'createdby' field is not being included?

Reply:
@JeyaSolirius - we are containing the createdby detail in the audit trail at the moment